runtime/README.md

```diff
@@ -58,7 +58,100 @@ Caliptra Runtime Firmware will share driver code with ROM and FMC where
 possible; however, it will have its own copies of all of these drivers linked into
 the Runtime Firmware binary.
 
-## Maibox commands
+## Manifest-Based Image Authorization (new in 1.2)
```

Comment:
This section should link the to auth manifest readme so the reader can easily find more details.